lib/mrsk/utils.rb

```diff
@@ -5,7 +5,7 @@ module Mrsk::Utils
   def argumentize(argument, attributes, redacted: false)
     Array(attributes).flat_map do |k, v|
       if v.present?
-        [ argument, redacted ? redact("#{k}=#{v}") : "#{k}=#{v}" ]
+        [ argument, redacted ? redact("#{k}=\"#{v}\"") : "#{k}=\"#{v}\"" ]
```

Comment:
Don't we need to escape any potential "'s inside the v?

Reply:
Yes thank you ! I've used the `dump` function which does exactly that, and added a gsub escape for backticks which are not escaped by this function.